### PR TITLE
Add TimeInstant + explicitAttr tests (func test 533)

### PR DESCRIPTION
**Pull request by AlvaroVega**
This is another  suggestion to fix cases proposed in https://github.com/telefonicaid/iotagent-node-lib/pull/1610

having into account this logic This is my suggestion to fix cases proposed in https://github.com/telefonicaid/iotagent-node-lib/pull/1610

- [x] update doc with new behavior about ```explicitAttrs = [ ]``` and ```timestmap = true```

### CODE REVIEW

File: lib/services/ngsi/entities-NGSI-v2.js [hunk 1]
```diff
@@ -521,7 +521,7 @@ function sendUpdateValueNgsi2(entityName, originMeasures, originTypeInformation,
         if (typeof typeInformation.explicitAttrs === 'string') {
```
Comment (fgalan) [inline lib/services/ngsi/entities-NGSI-v2.js:521]: If at the end this PR is going to be merged, an entry in CNR should be include.

In addition, let's check if documentation is properly describing this behaviour (in negative case, it should be modified)

Comment (AlvaroVega) [reply lib/services/ngsi/entities-NGSI-v2.js:521]: Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1618/commits/3176f6cf595a88350a0b25ecd1097fb14c2fe18f

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- Feature: do not propage TimeInstnat when explicitAttrs is empty array
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: do not propage TimeInstant when explicitAttrs is empty array (#1606)
```

File: doc/api.md [hunk 3]
```diff
@@ -464,7 +464,8 @@ element. By adding the field `explicitAttrs` with `true` value to device or conf
 measure elements that are not defined in the mappings of device or config group, persisting only the one defined in the
 mappings of the provision. If `explicitAttrs` is provided both at device and config group level, the device level takes
 precedence. Additionally `explicitAttrs` can be used to define which measures (identified by their attribute names, not
-by their object_id) defined in JSON/JEXL array will be propagated to NGSI interface.
+by their object_id) defined in JSON/JEXL array will be propagated to NGSI interface. When `explicitAttrs` is an array or
+and JEXL expression resulting in to Array, if this array is empty then `TimeInstant` is not propaged to CB.
```
Comment (fgalan) [inline doc/api.md:469]:
```suggestion
by their object_id) defined in JSON/JEXL array will be propagated to NGSI interface.

Note that when `explicitAttrs` is an array or a JEXL expression resulting in to Array, if this array is empty then `TimeInstant` is not propaged to CB.

```

Comment (fgalan) [reply doc/api.md:469]: I think is better in a separate paragraph to make it more relevant.

**Review by fgalan — APPROVED** [review]
LGTM

Passing ball to @mapedraza

**Review by mrutid — APPROVED** [review]
LGTM

**Pull request merged**